mne/viz/tests/test_3d.py

```diff
@@ -217,8 +220,17 @@ def test_plot_alignment(tmpdir, renderer, mixed_fwd_cov_evoked):
     renderer.backend._close_all()
     # no-head version
     renderer.backend._close_all()
+    # trans required
+    with pytest.raises(ValueError, match='transformation matrix is required'):
+        plot_alignment(info, trans=None, src=src_fname)
+    with pytest.raises(ValueError, match='transformation matrix is required'):
+        plot_alignment(info, trans=None, coord_frame='mri')
+    with pytest.raises(ValueError, match='transformation matrix is required'):
+        plot_alignment(info, trans=None, mri_fiducials=True)
+    with pytest.raises(ValueError, match='transformation matrix is required'):
+        plot_alignment(info, trans=None, surfaces=['brain'])
     # all coord frames
-    plot_alignment(info)  # works: surfaces='auto' default
+    plot_alignment(info, trans=trans_fname)  # works: surfaces='auto' default
```

Comment:
Should not be required, revert

Reply:
This is to suppress the warning...

Reply:
See my other comments, plotting sensors in `head` coordinate frame (the default for `plot_alignment`) `trans` should not be required, so having it be `None` is not a problem/no need to warn. I'm asking you to be more selective about when the warning is emitted...